Sources/XcodeGenKit/SourceGenerator.swift

```diff
@@ -400,7 +411,7 @@ class SourceGenerator {
 
         if let baseLocalisedDirectory = baseLocalisedDirectory {
             for filePath in try baseLocalisedDirectory.children()
-                .filter(isIncludedPath)
+                .filter({ self.isIncludedPath($0, targetSourceExcludePaths: targetSourceExcludePaths, targetSourceIncludePaths: targetSourceIncludePaths) })
```

Comment:
Nitpick: trailing closures

Reply:
I actually had to include the `()` because this is part of the `for in` statement. I just extracted the initialization of this sequence into its own line for clarity anyway.